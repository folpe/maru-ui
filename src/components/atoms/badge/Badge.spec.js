import React from 'react'
import { render } from '@testing-library/react'
import { Badge } from './'

describe('<Badge/>', () => {
  it('should otherwise match snapshot', () => {
    const { container } = render(<Badge />)
    expect(container).toMatchSnapshot()
  })
})
