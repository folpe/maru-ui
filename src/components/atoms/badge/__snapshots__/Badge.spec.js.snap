// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge/> should otherwise match snapshot 1`] = `
<div>
  <div
    class="badge circle primary medium"
  />
</div>
`;
